modules/axconfig/Cargo.toml

```diff
@@ -14,6 +14,7 @@ platform-pc-x86 = []
 platform-qemu-virt-riscv = []
 platform-qemu-virt-aarch64 = []
 platform-raspi4-aarch64 = []
+platform-bsta1000b-fada-aarch64 = []
```

Comment:
The platform name is too long, can it be simplified? Such as `platform-bsta1000b`

Reply:
all right. It named aarch64-bsta1000b